.github/workflows/ci.yml

```diff
@@ -21,13 +20,27 @@ jobs:
           MYSQL_USER: drupal
           MYSQL_PASSWORD: drupal
           MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
         options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=10s --health-retries=10
+      drupal:
+        image: phenaproxima/acquia_cms:headless
+        ports:
+          - 80:80
+        options: --name drupal
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
         node-version: [14.x, 16.x, 18.x]
     steps:
       - uses: actions/checkout@v3
+      - name: Import database
+        run: |
+          docker cp ./drupal/database.php.gz drupal:/opt/drupal/web
```

Comment:
Hi @phenaproxima I would suggest on not to store db dump (or any tests artifacts) in repo (unless it's really important). Though I noticed the size of file is very small. Because in the past, we used to do the same in Acquia CMS, which made the repo size large. And now we are pulling it on run-time i.e from s3 buckets.

Reply:
It's very small because it's gzipped. When expanded, it's a lot larger, although it's not TOO big because it doesn't contain any cache data.

It's absolutely critical to store the database dump, because the container does not do a full install of Acquia CMS from the ground up -- it's simply too long and involved of a process, and the entire point of containerizing it was to skip over that in favor of just importing it when we need it.

Reply:
Having 466kb DB dump in the repository is not ideal but I think we can figure out a better solution for this in future. I don't think we should block this PR on this given that this PR would improve our CI build times and stability.